components/scream/src/control/atmosphere_driver.cpp

```diff
@@ -395,107 +369,110 @@ void AtmosphereDriver::finalize ( /* inputs? */ ) {
   // Finalize output streams, make sure files are closed
   m_output_manager.finalize();
 
-  m_field_repo->clean_up();
-#ifdef SCREAM_DEBUG
-  m_bkp_field_repo.clean_up();
-#endif
+  for (auto it : m_field_repos) {
+    it.second->clean_up();
+  }
 
   if (scorpio::is_eam_pio_subsystem_inited()) {
     scorpio::eam_pio_finalize();
   }
 }
 
+AtmosphereDriver::field_repo_ptr
+AtmosphereDriver::get_field_repo () const {
```

Comment:
Since this is retrieving the field repo for the reference grid, do you think it would be more clear to call this `get_ref_field_repo`? I don't feel strongly about this. I just wondered how 
an AtmosphereDriver knew which field repo to use when I saw the first call.

Reply:
good point.